tests/integration/test_system_logs/test_system_logs.py

```diff
@@ -0,0 +1,73 @@
+# pylint: disable=line-too-long
+# pylint: disable=unused-argument
+# pylint: disable=redefined-outer-name
+
+import pytest
+from helpers.cluster import ClickHouseCluster
+
+cluster = ClickHouseCluster(__file__)
+
+node1 = cluster.add_instance(
+    "node1",
+    main_configs=["configs/config_order_by.xml"],
```

Comment:
Maybe I miss something but I don't find this file in this PR. On the other hand, a new file "system_logs_order_by.xml" was added but it was not referenced anywhere. Did you perhaps mix both files up?

Reply:
Ah, wrong indeed :(